src/OpenColorIO/transforms/LookTransform.cpp

```diff
@@ -327,12 +327,13 @@ void BuildLookOps(OpRcPtrVec & ops,
     looks.parse(lookTransform.getLooks());
 
     // We must handle the inverse src/dst colorspace transformation explicitly.
```

Comment:
Is this comment accurate for the new code? (What does "handle explicitly" mean?)

Reply:
Good question Dennis.  The comment is from v1 and I'm not sure what it meant either.  In any case, I don't think Remi's change affects the validity of it.  The lookTransform direction was being ignored before Remi's change, which is clearly a bug.

Reply:
I understood that as a just a reminder that src and dst colorspace needs to be swapped, but I may have misunderstood the purpose of this comment, I guess it could be removed if it introduce more confusion that anything.

Reply:
I also understand the comment as a reminder. I would only change the beginning of the sentence to `The code must handle ...`

Reply:
Fixed